WikiSentiment/AzureStorageClient.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Azure;
+using Azure.Data.Tables;
+
+namespace WikiSentiment
+{
+    /// <summary>
+    /// API wrapper for azure table storage
+    /// </summary>
+    public class AzureStorageClient:IDBClient
+    {
+        TableClient client;
+
+        public AzureStorageClient(TableClient _client)
+        {
+            client = _client;
+        }
+
+        /// <summary>
+        /// Attempt to get data for given date, returns empty string if none exists
+        /// </summary>
+        /// <param name="_YYYYMM"></param>
+        /// <param name="_DD"></param>
+        /// <returns></returns>
+        public async Task<string> Load(DateTime _date)
+        {
+            //Format is YYYY-MM as pkey, DD as rkey
+            AsyncPageable<DayEntity> result = client.QueryAsync<DayEntity>(
+                e => e.PartitionKey == $"{ _date.Year:D4}-{_date.Month:D2}" && e.RowKey == $"{_date.Day:D2}");
+
+            await foreach (var res in result)
```

Comment:
```suggestion
            return await result.FirstOrDefault()
```
Maybe?. I'm relatively unfamiliar with async code.